lib/main_common.pm

```diff
@@ -2387,10 +2387,11 @@ sub set_mu_virt_vars {
     $BUILD =~ /^:(\d+):([^:]+)$/im;
 
     die "BUILD value is $BUILD, but does not match required format." if (!$2);
-
     my $_pkg = $2;
     my $_update_package = '';
-    if ($_pkg =~ /qemu|xen|virt-manager|libguestfs|libslirp|open-vm-tools/) {
+    if ($_pkg =~ /none/) {
```

Comment:
Will be good to add a comment to explain purpose of none -- facilitate devel test with no real incident?